packages/dependency-operator-filter/contracts/mock/MockMarketPlace1.sol

```diff
@@ -65,7 +65,8 @@ contract MockERC1155MarketPlace1 is ERC1155Receiver, ERC721Holder {
         return ERC1155_BATCH_RECEIVED;
     }
 
-    function supportsInterface(bytes4 _interfaceId) public view override returns (bool) {
-        super.supportsInterface(_interfaceId);
+    function supportsInterface(bytes4 _interfaceId) public view override returns (bool interfaceId) {
+        interfaceId = super.supportsInterface(_interfaceId);
+        return interfaceId;
```

Comment:
Why do it like that? I mean these are mocks so there is no harm but just as a suggestion to write less code:

You could do:

```
function supportsInterface(bytes4 _interfaceId) public view override returns (bool interfaceId) {
        interfaceId = super.supportsInterface(_interfaceId);
}
```
or
```
function supportsInterface(bytes4 _interfaceId) public view override returns (bool interfaceId) {
        return super.supportsInterface(_interfaceId);
}
```
or

```
function supportsInterface(bytes4 _interfaceId) public view override returns (bool) {
        return super.supportsInterface(_interfaceId);
}
```

Reply:
This resolves the warning of un-nammed return value

Reply:
The above two also solve that issue and with a single line of code, but its fine since those are mocks anyway